src/sentry/static/sentry/app/stores/configStore.tsx

```diff
@@ -2,28 +2,29 @@ import moment from 'moment-timezone';
 import Reflux from 'reflux';
 import qs from 'query-string';
 import {setLocale} from 'app/locale';
+import {Config} from 'app/types';
 
 const ConfigStore = Reflux.createStore({
-  init() {
+  init(): void {
```

Comment:
@lynnagara Isn't the `void` determined implicitly?

Reply:
Yeah, was just being explicit. Can omit if if preferred.

Reply:
@lynnagara Let me ask on Slack...

Reply:
IMO, being explicit is the better approach. 

My reasoning is that the reason why we write out some type annotations in the first place is to help out the TypeScript compiler by providing better type hints.